js/interface.js

```diff
@@ -2555,8 +2569,21 @@ $('[data-save-rule]').click(function(event) {
 
   if (currentDataSourceRuleIndex === undefined) {
     currentDataSourceRules.push(rule);
+
+    // For Add and Edit security rule, adding the rule in current final rules
+    var modalTitle = $('#configure-rule').find('.modal-title').text();
```

Comment:
Assuming there's only 2 types of title, this is sufficient and reduces the number of static text to just 1.

Also, this should be defined outside the `if` clause so it can be re-used in the `else` clause.

```suggestion
    var isAddingRule = $('#configure-rule').find('.modal-title').text() === 'Add new security rule';
```

Reply:
Did the changes as per you suggestion.